src/App.js

```diff
@@ -49,16 +50,17 @@ class App extends React.Component {
         <div className="main-App">
           <div className="form-container">
             <Form getWeather={this.getWeather} />
+            <div className="error"> {this.state.error}</div>
           </div>
 
-          <div class-name="forecast-container">
+          <div class-name="forecast-container0">
             {this.state.weatherData && (
               <Weather currentData={this.state.weatherData.list[0]} />
             )}
           </div>
         </div>
         <div class-name="forecast-container">
-          {this.state.weatherData && <ForeCast data={this.state.weatherData} />}
+          {this.state.weatherData && <ForeCast data={this.state.data} />}
```

Comment:
As per my comment above, you can just pass `{this.state.weatherData}` to the `Forecast` component.
This won't affect the fact that you are passing `{this.state.weatherData.list[0]}` to the `Weather` component. You're not modifying the data when you pass it, you're just passing different parts of it to different components, so you can use the same variable (weatherData) in both cases.